### PR TITLE
Automation JWT status tests

### PR DESCRIPTION
**Pull request by Avi-Eyal**
Added some JWT status automation tests: 

- valid status check
- signing key is not configured
- signing key is badly configured (jwks and provider both configured)
- user is not permitted on status webservice

### CODE REVIEW

File: cucumber/authenticators_jwt/features/step_definitions/authn_jwt_steps.rb [hunk 1]
```diff
@@ -6,6 +6,10 @@
   create_jwt_secret(variable, value)
 end
 
+Given(/I successfully set authn-jwt provider-uri variable with value of "([^"]*)" endpoint/) do |provider_uri|
```
Comment (tzheleznyak) [inline cucumber/authenticators_jwt/features/step_definitions/authn_jwt_steps.rb:9]: Delete this step and use the one in line 5 here

File: cucumber/authenticators_jwt/features/authn_status_jwt.feature [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+    - !user alice
+    """
+    And I am the super-user
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+    And I successfully set authn-jwt "token-app-property" variable to value "user"
+    And I login as "alice"
+    And I save my place in the log file
+    When I GET "/authn-jwt/raw/cucumber/status"
+    Then the HTTP response status code is 403
+    And the authenticator status check fails with error "CONJ00006E 'alice' does not have 'read' privilege on cucumber:webservice:conjur/authn-jwt/raw/status"
```
Comment (tzheleznyak) [inline cucumber/authenticators_jwt/features/authn_status_jwt.feature:213]: Add empty line

File: cucumber/authenticators_jwt/features/step_definitions/authn_jwt_steps.rb (hunk 1 above)
Comment (Avi-Eyal) [reply cucumber/authenticators_jwt/features/step_definitions/authn_jwt_steps.rb:9]: nice catch

**Review by tzheleznyak — APPROVED** [review]
LGTM

**Review by eladkug — APPROVED** [review]

**Review by tzheleznyak — APPROVED** [review]

**Pull request merged**